xbmc/video/VideoInfoTag.cpp

```diff
@@ -604,6 +649,13 @@ void CVideoInfoTag::ToSortable(SortItem& sortable, Field field) const
   }
 }
 
+const CRating CVideoInfoTag::GetRating(std::string type) const
+{
+  if (type.empty())
+    return m_ratings.find(m_strDefaultRating)->second;
```

Comment:
I'd rather call `GetRating()` again with `m_strDefaultRating` to avoid code duplication.